### ISSUE TITLE
[Bug]: Honkai: Star Rail, Inconsistently Global Graphics Settings: Character Settings in game High, but in Collapse is Very High

### ISSUE THREAD

**Issue by IRedDragonICY** [CollapseLauncher/Collapse#399]
### Affected Version

Collapse 1.73.3 Preview

### Bug Behavior

[DxDiag.txt](https://github.com/CollapseLauncher/Collapse/files/14231870/DxDiag.txt)

### Expected Behavior

Add selection on Character Settings in game
1. Low
2. Medium
3. High

If we select High, it must **High** to in Global Graphics Settings: Character Settings 

### Steps to reproduce

Change In game settings into Very high
The max Character Quality is **high**
after close Honkai: Star Rail, save settings in Collapse not same as in game Graphics Settings

### Screenshot(s)

In game
![image](https://github.com/CollapseLauncher/Collapse/assets/98265563/aaa6b8dc-3f5b-4502-a095-c1e891768404)
Collapse Launcher Settings
![image](https://github.com/CollapseLauncher/Collapse/assets/98265563/d5b2f018-e16c-4c52-81c7-422b798e343e)

**Comment by bagusnl** [comment]
Heya!

Thank you for submitting bug report for Collapse~ I have sent a fix commit on https://github.com/CollapseLauncher/Collapse/pull/398/commits/e228c5534d70e4e137b4d0d03d3872caa1425fa7 
As the fix is under #398 it might need to take a bit until it gets merged into main and hopefully we can include the fix on next preview version !

**Comment by neon-nyan** [comment]
Hi there,
This issue has been fixed as per latest update, 1.73.4 for both Stable and Preview releases.
Please make sure you have your launcher up-to-date to receive the fix and some new features.

Closing this issue for now and feel free to reopen the issue if you feel that the issue hasn't been resolved.

**Comment by IRedDragonICY** [comment]
@neon-nyan the bug occurs again
![image](https://github.com/CollapseLauncher/Collapse/assets/98265563/55554baf-1939-46ef-bf0d-778c8981a5f8)
Graphics Settings
![image](https://github.com/CollapseLauncher/Collapse/assets/98265563/0727ea6e-c1a2-4023-bf0e-be5311c0959a)
![image](https://github.com/CollapseLauncher/Collapse/assets/98265563/45d2c6f2-719f-4bc2-b548-ae6ffa4e564d)
it's show very high, but Game Option is High

**Comment by bagusnl** [comment]
Will investigate, thanks for the heads up!

**Comment by bagusnl** [comment]
After investigating I can't seem to trigger the same bug, tho I'll make a clamp so it will only show max High on Collapse

**Comment by shatyuka** [comment]
Almost forgot about this XD. Can be reproduced here (or rather, never worked).

**Comment by shatyuka** [comment]
@IRedDragonICY Can you provide the registry of game graphics settings?
Run the following command in PowerShell:
```powershell
[Text.Encoding]::UTF8.GetString((Get-ItemPropertyValue "HKCU:\Software\miHoYo\Star Rail" GraphicsSettings_Model_h2986158309))
```

**Comment by shatyuka** [comment]
Try this CI build: https://github.com/CollapseLauncher/Collapse/actions/runs/9050660614

**Comment by IRedDragonICY** [comment]
> ```powershell
> [Text.Encoding]::UTF8.GetString((Get-ItemPropertyValue "HKCU:\Software\miHoYo\Star Rail" GraphicsSettings_Model_h2986158309))
> ```

uhm different regedit location so Powershell script is doesn't work
Computer\HKEY_CURRENT_USER\Software\Cognosphere\Star Rail
```powershell
$path = "HKCU:\Software\Cognosphere\Star Rail"
$name = "GraphicsSettings_Model_h2986158309"

try {
    $value = Get-ItemProperty -Path $path -Name $name
    $binaryData = $value."$name"
    $stringData = [System.Text.Encoding]::UTF8.GetString($binaryData)
    $stringData
} catch {
    Write-Host "not found"
}

```
here is settings:
''
7B22465053223A36302C22456E61626C655653796E63223A66616C73652C2252656E6465725363616C65223A312E382C225265736F6C7574696F6E5175616C697479223A352C22536861646F775175616C697479223A352C224C696768745175616C697479223A352C224368617261637465725175616C697479223A352C22456E7644657461696C5175616C697479223A352C225265666C656374696F6E5175616C697479223A352C225346585175616C697479223A342C22426C6F6F6D5175616C697479223A352C2241414D6F6465223A312C22456E61626C654D6574616C46585355223A66616C73657D00
''

``
{"FPS":60,"EnableVSync":false,"RenderScale":1.8,"ResolutionQuality":5,"ShadowQuality":5,"LightQuality":5,"CharacterQuality":5,"EnvDetailQuality":5,"ReflectionQuality":5,"SFXQuality":4,"BloomQuality":5,"AAMode":1,"EnableMetalFXSU":false}
``

**Comment by IRedDragonICY** [comment]
> Try this CI build: https://github.com/CollapseLauncher/Collapse/actions/runs/9050660614

it's fixed in this build

**Comment by bagusnl** [comment]
An update that include this fix already been published on 1.80.5, we'll close this issue ticket.
If you have any problem, don't hesitate to reopen this or make a new ticket.

Thank you for using Collapse Launcher <3
